modules/op-approving-task-web-console.adoc

```diff
@@ -0,0 +1,28 @@
+// This module is included in the following assemblies:
+// * create/using-manual-approval.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="op-approving-task-web-console_{context}"]
+= Approving a manual approval task by using the web console
+
+You can approve or reject a manual approval task by using the {OCP} web console.
+
+If you are listed as an approver in a manual approval task and a pipeline run reaches this task, the web console a notification. You can view a list of tasks that require your approval and approve or reject these tasks.
+
+.Prerequisites
+
+* You enabled the {pipelines-shortname} console plugin.
+
+.Procedure
+
+. View a list of tasks that you can approve by completing one of the following actions:
+** When a notification about a task requiring your approval displays, click *Go to Approvals tab* in this notification.
+** In the *Administrator* perspective menu, select *Pipelines* -> *Pipelines* and then click the *Approvals* tab.
+** In the *Developer* perspective menu, select *Pipelines* and then click the *Approvals* tab.
+** In the *PipelineRun details* window, in the *Details* tab, click the rectangle that represents the manual approval task. The list displays only the approval for this task.
+** In the *PipelineRun details* window, click the *ApprovalTasks* tab. The list displays only the approval for this pipeline run.
+. In the list of aproval tasks, in the line that represents the task that you want to approve, click the {kebab} icon and then select one of the following options:
+** To approve the task, select *Approve*.
+** To reject the task, select *Reject*.
+. Enter a message in the *Reason* field.
```

Comment:
IMHO this should be optional, should we stress that fact?

Reply:
I agree this should be optional, but it is not optional. The Submit button was disabled until I entered something in the Reason field.